scrambler.cpp

```diff
@@ -944,6 +955,13 @@ int main(int argc, char **argv)
                 support_z3 = true;
             } else if (strcmp(argv[i + 1], "false") == 0) {
                 support_z3 = false;
+            }
```

Comment:
the `else { usage(arv[0))` is missing here.

Reply:
Done.